package.json

```diff
@@ -8,12 +8,16 @@
     "build": "graph build",
     "create-local": "graph create example --node http://127.0.0.1:8020",
     "deploy-local": "graph deploy example --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
-    "test": "graph test gravity"
+    "test": "tsc --traceResolution --noEmit --skipLibCheck && graph test gravity"
   },
   "dependencies": {
     "@graphprotocol/graph-cli": "^0.22.1",
     "@graphprotocol/graph-ts": "^0.22.1",
-    "matchstick-as": "git://github.com/LimeChain/matchstick-as.git#b9d6988a07923b8286d8c38ef25581368570bc2a",
```

Comment:
We need that exact version of matchstick-as since some of its content is still not released and is therefore not in version 0.2.0

Reply:
Actually the commit hash we need is - #c2d83fd1f661cd5a773ef0b98d8c8a5d86c68a8e